[2024-03-26 15:16:31,008] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.download_dataset_task scheduled__2022-07-02T06:00:00+00:00 [queued]>
[2024-03-26 15:16:31,041] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.download_dataset_task scheduled__2022-07-02T06:00:00+00:00 [queued]>
[2024-03-26 15:16:31,041] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-03-26 15:16:31,041] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2024-03-26 15:16:31,041] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-03-26 15:16:31,071] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2022-07-02 06:00:00+00:00
[2024-03-26 15:16:31,078] {standard_task_runner.py:52} INFO - Started process 203 to run task
[2024-03-26 15:16:31,091] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ingest_data', 'download_dataset_task', 'scheduled__2022-07-02T06:00:00+00:00', '--job-id', '241', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_dag.py', '--cfg-path', '/tmp/tmp_slmfm5i', '--error-file', '/tmp/tmpnbtpswj4']
[2024-03-26 15:16:31,103] {standard_task_runner.py:77} INFO - Job 241: Subtask download_dataset_task
[2024-03-26 15:16:31,219] {logging_mixin.py:109} INFO - Running <TaskInstance: ingest_data.download_dataset_task scheduled__2022-07-02T06:00:00+00:00 [running]> on host 153a478eba08
[2024-03-26 15:16:31,283] {warnings.py:99} WARNING - /home/***/.local/lib/python3.6/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-03-26 15:16:31,319] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ingest_data
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-07-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-02T06:00:00+00:00
[2024-03-26 15:16:31,320] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2024-03-26 15:16:31,321] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sSL <***.models.param.DagParam object at 0x7f7765472518> > /opt/***/raw_data/yellowtaxi _tripdata_2022-07.csv']
[2024-03-26 15:16:31,337] {subprocess.py:85} INFO - Output:
[2024-03-26 15:16:31,341] {subprocess.py:89} INFO - bash: -c: line 0: syntax error near unexpected token `>'
[2024-03-26 15:16:31,342] {subprocess.py:89} INFO - bash: -c: line 0: `curl -sSL <***.models.param.DagParam object at 0x7f7765472518> > /opt/***/raw_data/yellowtaxi _tripdata_2022-07.csv'
[2024-03-26 15:16:31,342] {subprocess.py:93} INFO - Command exited with return code 1
[2024-03-26 15:16:31,363] {taskinstance.py:1700} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1329, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1455, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1511, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/bash.py", line 188, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-03-26 15:16:31,373] {taskinstance.py:1277} INFO - Marking task as FAILED. dag_id=ingest_data, task_id=download_dataset_task, execution_date=20220702T060000, start_date=20240326T151631, end_date=20240326T151631
[2024-03-26 15:16:31,398] {standard_task_runner.py:92} ERROR - Failed to execute job 241 for task download_dataset_task
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 298, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1329, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1455, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1511, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/operators/bash.py", line 188, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-03-26 15:16:31,429] {local_task_job.py:154} INFO - Task exited with return code 1
[2024-03-26 15:16:31,493] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-26 17:33:55,970] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.download_dataset_task scheduled__2022-07-02T06:00:00+00:00 [queued]>
[2024-03-26 17:33:55,993] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.download_dataset_task scheduled__2022-07-02T06:00:00+00:00 [queued]>
[2024-03-26 17:33:55,994] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-03-26 17:33:55,994] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2024-03-26 17:33:55,994] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-03-26 17:33:56,020] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2022-07-02 06:00:00+00:00
[2024-03-26 17:33:56,032] {standard_task_runner.py:52} INFO - Started process 2858 to run task
[2024-03-26 17:33:56,048] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ingest_data', 'download_dataset_task', 'scheduled__2022-07-02T06:00:00+00:00', '--job-id', '416', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_dag.py', '--cfg-path', '/tmp/tmpv27yh9j6', '--error-file', '/tmp/tmpsw0yv_ga']
[2024-03-26 17:33:56,049] {standard_task_runner.py:77} INFO - Job 416: Subtask download_dataset_task
[2024-03-26 17:33:56,194] {logging_mixin.py:109} INFO - Running <TaskInstance: ingest_data.download_dataset_task scheduled__2022-07-02T06:00:00+00:00 [running]> on host d71984e59113
[2024-03-26 17:33:56,257] {warnings.py:99} WARNING - /home/***/.local/lib/python3.6/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-03-26 17:33:56,289] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ingest_data
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-07-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-02T06:00:00+00:00
[2024-03-26 17:33:56,291] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2024-03-26 17:33:56,292] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sSL https://d37ci6vzurychx.cloudfront.net/trip-data/yellow_tripdata_2022-07.parquet > /opt/***/yellowtaxi_tripdata_2022-07.parquet']
[2024-03-26 17:33:56,313] {subprocess.py:85} INFO - Output:
[2024-03-26 17:34:02,297] {subprocess.py:93} INFO - Command exited with return code 0
[2024-03-26 17:34:02,408] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ingest_data, task_id=download_dataset_task, execution_date=20220702T060000, start_date=20240326T173355, end_date=20240326T173402
[2024-03-26 17:34:02,521] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-03-26 17:34:02,576] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-03-26 17:48:48,882] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.download_dataset_task scheduled__2022-07-02T06:00:00+00:00 [queued]>
[2024-03-26 17:48:48,983] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.download_dataset_task scheduled__2022-07-02T06:00:00+00:00 [queued]>
[2024-03-26 17:48:48,983] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-03-26 17:48:48,983] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2024-03-26 17:48:48,983] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-03-26 17:48:49,071] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2022-07-02 06:00:00+00:00
[2024-03-26 17:48:49,084] {standard_task_runner.py:52} INFO - Started process 276 to run task
[2024-03-26 17:48:49,109] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ingest_data', 'download_dataset_task', 'scheduled__2022-07-02T06:00:00+00:00', '--job-id', '497', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_dag.py', '--cfg-path', '/tmp/tmpajxzewt_', '--error-file', '/tmp/tmp_2uqmiv5']
[2024-03-26 17:48:49,109] {standard_task_runner.py:77} INFO - Job 497: Subtask download_dataset_task
[2024-03-26 17:48:49,301] {logging_mixin.py:109} INFO - Running <TaskInstance: ingest_data.download_dataset_task scheduled__2022-07-02T06:00:00+00:00 [running]> on host 05f84def3a12
[2024-03-26 17:48:49,413] {warnings.py:99} WARNING - /home/***/.local/lib/python3.6/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-03-26 17:48:49,462] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ingest_data
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-07-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-02T06:00:00+00:00
[2024-03-26 17:48:49,464] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2024-03-26 17:48:49,464] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sSL https://d37ci6vzurychx.cloudfront.net/trip-data/yellow_tripdata_2022-07.parquet > /opt/***/yellowtaxi_tripdata_2022-07.parquet']
[2024-03-26 17:48:49,536] {subprocess.py:85} INFO - Output:
[2024-03-26 17:48:55,755] {subprocess.py:93} INFO - Command exited with return code 0
[2024-03-26 17:48:56,618] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ingest_data, task_id=download_dataset_task, execution_date=20220702T060000, start_date=20240326T174848, end_date=20240326T174856
[2024-03-26 17:48:56,850] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-03-26 17:48:57,101] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-03-26 17:52:20,619] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.download_dataset_task scheduled__2022-07-02T06:00:00+00:00 [queued]>
[2024-03-26 17:52:20,645] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.download_dataset_task scheduled__2022-07-02T06:00:00+00:00 [queued]>
[2024-03-26 17:52:20,645] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-03-26 17:52:20,646] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2024-03-26 17:52:20,646] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-03-26 17:52:20,696] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2022-07-02 06:00:00+00:00
[2024-03-26 17:52:20,731] {standard_task_runner.py:52} INFO - Started process 686 to run task
[2024-03-26 17:52:20,743] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ingest_data', 'download_dataset_task', 'scheduled__2022-07-02T06:00:00+00:00', '--job-id', '573', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_dag.py', '--cfg-path', '/tmp/tmph_l9h2eg', '--error-file', '/tmp/tmp3oy5wtwn']
[2024-03-26 17:52:20,744] {standard_task_runner.py:77} INFO - Job 573: Subtask download_dataset_task
[2024-03-26 17:52:20,864] {logging_mixin.py:109} INFO - Running <TaskInstance: ingest_data.download_dataset_task scheduled__2022-07-02T06:00:00+00:00 [running]> on host 05f84def3a12
[2024-03-26 17:52:20,940] {warnings.py:99} WARNING - /home/***/.local/lib/python3.6/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-03-26 17:52:20,996] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ingest_data
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-07-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-02T06:00:00+00:00
[2024-03-26 17:52:20,997] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2024-03-26 17:52:20,998] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sSL https://d37ci6vzurychx.cloudfront.net/trip-data/yellow_tripdata_2022-07.parquet > /opt/***/yellowtaxi_tripdata_2022-07.parquet']
[2024-03-26 17:52:21,037] {subprocess.py:85} INFO - Output:
[2024-03-26 17:52:27,412] {subprocess.py:93} INFO - Command exited with return code 0
[2024-03-26 17:52:27,598] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ingest_data, task_id=download_dataset_task, execution_date=20220702T060000, start_date=20240326T175220, end_date=20240326T175227
[2024-03-26 17:52:27,828] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-03-26 17:52:28,552] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-26 18:04:56,642] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.download_dataset_task scheduled__2022-07-02T06:00:00+00:00 [queued]>
[2024-03-26 18:04:56,789] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.download_dataset_task scheduled__2022-07-02T06:00:00+00:00 [queued]>
[2024-03-26 18:04:56,789] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-03-26 18:04:56,789] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2024-03-26 18:04:56,794] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-03-26 18:04:56,933] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2022-07-02 06:00:00+00:00
[2024-03-26 18:04:57,013] {standard_task_runner.py:52} INFO - Started process 1634 to run task
[2024-03-26 18:04:57,047] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ingest_data', 'download_dataset_task', 'scheduled__2022-07-02T06:00:00+00:00', '--job-id', '651', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_dag.py', '--cfg-path', '/tmp/tmpm0fvyp6m', '--error-file', '/tmp/tmpidyqbgiv']
[2024-03-26 18:04:57,084] {standard_task_runner.py:77} INFO - Job 651: Subtask download_dataset_task
[2024-03-26 18:04:57,474] {logging_mixin.py:109} INFO - Running <TaskInstance: ingest_data.download_dataset_task scheduled__2022-07-02T06:00:00+00:00 [running]> on host 05f84def3a12
[2024-03-26 18:04:57,921] {warnings.py:99} WARNING - /home/***/.local/lib/python3.6/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-03-26 18:04:58,297] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ingest_data
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-07-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-02T06:00:00+00:00
[2024-03-26 18:04:58,298] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2024-03-26 18:04:58,299] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sSL https://d37ci6vzurychx.cloudfront.net/trip-data/yellow_tripdata_2022-07.parquet > /opt/***/yellowtaxi_tripdata_2022-07.parquet']
[2024-03-26 18:04:58,649] {subprocess.py:85} INFO - Output:
[2024-03-27 03:48:04,635] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.download_dataset_task scheduled__2022-07-02T06:00:00+00:00 [queued]>
[2024-03-27 03:48:04,677] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.download_dataset_task scheduled__2022-07-02T06:00:00+00:00 [queued]>
[2024-03-27 03:48:04,677] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-03-27 03:48:04,677] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2024-03-27 03:48:04,677] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-03-27 03:48:04,705] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2022-07-02 06:00:00+00:00
[2024-03-27 03:48:04,720] {standard_task_runner.py:52} INFO - Started process 100 to run task
[2024-03-27 03:48:04,728] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ingest_data', 'download_dataset_task', 'scheduled__2022-07-02T06:00:00+00:00', '--job-id', '758', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_dag.py', '--cfg-path', '/tmp/tmp67_wn1nx', '--error-file', '/tmp/tmpl8l3kzjm']
[2024-03-27 03:48:04,728] {standard_task_runner.py:77} INFO - Job 758: Subtask download_dataset_task
[2024-03-27 03:48:04,857] {logging_mixin.py:109} INFO - Running <TaskInstance: ingest_data.download_dataset_task scheduled__2022-07-02T06:00:00+00:00 [running]> on host 05f84def3a12
[2024-03-27 03:48:04,928] {warnings.py:99} WARNING - /home/***/.local/lib/python3.6/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-03-27 03:48:04,987] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ingest_data
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-07-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-02T06:00:00+00:00
[2024-03-27 03:48:04,990] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2024-03-27 03:48:04,991] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sSL https://d37ci6vzurychx.cloudfront.net/trip-data/yellow_tripdata_2022-07.parquet > /opt/***/yellowtaxi_tripdata_2022-07.parquet']
[2024-03-27 03:48:05,017] {subprocess.py:85} INFO - Output:
[2024-03-27 03:48:11,143] {subprocess.py:93} INFO - Command exited with return code 0
[2024-03-27 03:48:11,192] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ingest_data, task_id=download_dataset_task, execution_date=20220702T060000, start_date=20240327T034804, end_date=20240327T034811
[2024-03-27 03:48:11,244] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-03-27 03:48:11,567] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
